components/twitter-card.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {fallbackLocale} from './locale'
+
+function TwitterCard({
+  card = 'summary_large_image',
+  title = 'Maintainable CSS with React',
+  description = 'React component styling solved with an elegant API, small footprint, and great performance',
+  pathname,
+  locale,
+}) {
+  const urlPrefix = fallbackLocale === locale ? '' : `${locale}.`
+  const image = `https://${urlPrefix}glamorous.rocks/static/images/banner.png`
```

Comment:
This needs to work with SSR for Twitter to pick up the URLs, so I think we need to use `req`.

Another option is to use a universal config and pass that in as the base URL. So we'd get something like `https://${prefix}${BASE_URL}${pathname}`

https://github.com/zeit/next.js/tree/master/examples/with-universal-configuration